client/extensions/wp-super-cache/cache-stats.jsx

```diff
@@ -56,23 +86,49 @@ class CacheStats extends Component {
 						</tr>
 					</thead>
 					<tbody>
-						{ map( files, ( file, dir ) => (
-							<tr className="wp-super-cache__stat" key={ dir }>
-								<td className="wp-super-cache__stat-dir">{ dir }</td>
-								<td>{ file.files }</td>
-								<td className="wp-super-cache__stat-age">{ getAge( file ) }</td>
-								<td className="wp-super-cache__stat-action">
-									<Button compact>
-										{ translate( 'Delete' ) }
-									</Button>
-								</td>
-							</tr>
-						) ) }
+						{ map( files, ( { files: count, lower_age, upper_age }, url ) => (
+						<tr className="wp-super-cache__stat" key={ url }>
+							<td className="wp-super-cache__stat-dir">
+								{ url }
+							</td>
+							<td>
+								{ count }
+							</td>
+							<td className="wp-super-cache__stat-age">
+								{ getAge( lower_age, upper_age ) }
+							</td>
+							<td className="wp-super-cache__stat-action">
+								<Button
+									compact
+									busy={ isDeleting && ( this.state.url === url ) }
+									data-url={ url }
```

Comment:
Hmm, OTOH, [this](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md#lists-of-items) might be a valid reason to leave as-is.